specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Migrate_away_from_BinaryFormatter.cs

```diff
@@ -0,0 +1,19 @@
+namespace Rules.MS_Build.Migrate_away_from_BinaryFormatter;
+
+public class Reports
+{
+    [Test]
+    public void when_BinaryFormatter_is_enabled() => new MigrateAwayFromBinaryFormatter()
+        .ForProject("BinaryFormattingEnabled.cs")
+        .HasIssue(Issue.WRN("Proj0032", "Migrate away from BinaryFormatter and disable <EnableUnsafeBinaryFormatterSerialization>.")
```

Comment:
The .csproj under tests initially enables, then later disables this flag, so one could argue it shouldn't report this options.

Reply:
There is no reason to enable it in the first place, so I'll report on setting it to true. I've even considered to report on setting it to false too due to that reasoning.